beta-src/src/components/ui/WDMoveControls.tsx

```diff
@@ -95,13 +99,20 @@ const WDMoveControls: React.FC<WDMoveControlsProps> = function ({
     }
   };
 
+  const commandActions = {
+    SAVE_ORDERS: (command) => {
+      clickButton(Move.SAVE);
+    },
+  };
```

Comment:
I think you need to delete the command after it is processed. Look at the command actions in WDTerritory, WDMapController, or WDUnitController as examples.